README.md

```diff
@@ -90,7 +90,7 @@ This [blog post](http://www.yegor256.com/2015/03/22/takes-java-web-framework.htm
 - [How to contribute](#how-to-contribute)
 - [Got questions?](#got-questions)
 
-## Quick Start
+## Quick Start[⬆](#contents)<!-- Link generated with jump2header -->
```

Comment:
@strdr4605 Do we have to keep these comments?

Reply:
@andreoss No, it's there only for info purpose for others to understand.